File: 🟢 Easy/217 - Contains Duplicate/_217ContainsDpulicate.cpp
class Solution {
public:
    bool containsDuplicate(vector<int>& nums) {
        sort(nums.begin(), nums.end());
        int val = nums[0];
        for (int i = 1; i < nums.size(); i++) {
            if (val == nums[i]) {
                return true;
            } else {
                val = nums[i];
            }
        }
        return false;
    }
};

File: 🟢 Easy/217 - Contains Duplicate/_217ContainsDpulicate.js
/**
 * @param {number[]} nums
 * @return {boolean}
 */
var containsDuplicate = function (nums) {
    nums.sort();
    let val = nums[0];
    for (let i = 1; i < nums.length; i++) {
        if (val == nums[i]) {
            return true;
        } else {
            val = nums[i];
        }
    }
    return false;
};